src/components/ProgressIndicators/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./StepBar";
```

Comment:
Put `StepBar` in it's own folder in `components`, rather than nesting it in `ProgressIndicators`